glmnet/linear.py

```diff
@@ -137,7 +137,7 @@ def __init__(self, alpha=1, n_lambda=100, min_lambda_ratio=1e-4,
         self.random_state = random_state
         self.verbose = verbose
 
-    def fit(self, X, y, sample_weight=None):
+    def fit(self, X, y, sample_weight=None, relative_penalties=None):
```

Comment:
The docstring below will need an entry for `relative_penalties`, looks like it's missing one for `sample_weight` too.